ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.17-nightly-etcd-chaos.yaml

```diff
@@ -0,0 +1,48 @@
+base_images:
+  cerberus.prow:
+    name: cerberus
+    namespace: chaos
+    tag: cerberus-prow
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: cerberus/Dockerfile_prow
+  from: cerberus.prow
+  to: cerberus-main-prow-417
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: krkn-hub-etcd-tests-417
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws-chaos
+    env:
+      BASE_DOMAIN: aws.rhperfscale.org
+      SPOT_INSTANCES: "true"
+      TELEMETRY_GROUP: prow-ocp-4.17-etcd-component-readiness
+    observers:
+      enable:
+      - redhat-chaos-cerberus
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: create-infra-move-ingress-monitoring-registry
```

Comment:
thoughts on just putting this a "4.17" yaml file since we'll be adding a lot of tests think having in multiple files will be too much. Is the name of the test telling enough for the dashboarding/result tracking to figure out

Reply:
Makes sense, updated the job name to component readiness. Thanks.
About the test name, updated it to include "component readiness", we can tweak it once we understand the requirements on the sippy side. Thoughts @paigerube14?